### PR TITLE
Release preparation: 5.9.0-rc.1

### PR DESCRIPTION
**Pull request by Konamiman**
### Changes proposed in this Pull Request:

* Add new entries for WooCommerce, Admin and Blocks to `changelog.txt`; and copy them to `readme.txt`

### Important!

After this is merged and cherrypicked into `release/5.9`, that branch needs to get an extra commit that:

* Changes `Version` in `woocommerce.php` to `5.9.0-RC.1`

Extra commit: https://github.com/woocommerce/woocommerce/commit/315d9c4c9ed996c6b115109fdd0086e6e5ffa6c4

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+* Fix - wc_get_price_excluding_tax when an order with no customer is passed. #31015
+* Fix - Rename transient used to cache data for Featured page of In-App Marketplace. #31002
+* Tweak - Add promoted cards styling to marketplace section. #30861
+
+**WooCommerce Admin - 2.8.0 RC 3**
+
+* Fix - Issue where stock activity panel was not rendering correctly. #7817
+* Fix - Increase CSS specificity to avoid conflicts and broken panel styling. #7813
+* Fix - Updated link to WooCommerce Developers Blog in readme.txt. #7824
```
Comment (peterfabian) [inline changelog.txt:15]: Aren't there more PRs merged to rc 3?

https://github.com/woocommerce/woocommerce-admin/compare/release/2.8.0-rc.2...release/2.8.0-rc.3

**Review by peterfabian — CHANGES_REQUESTED** [review]
Please check WC Admin chages. Thx!

**Review by peterfabian — APPROVED** [review]
I can see now it's based on https://github.com/woocommerce/woocommerce/pull/31018 so should he fine 👍

**Comment by Konamiman** [comment]
PR to add the changelog for 5.9 into `trunk` for 6.0: https://github.com/woocommerce/woocommerce/pull/31092

**Pull request merged**